src/pages/patterns/disclosures-pattern/index.mdx

```diff
@@ -0,0 +1,779 @@
+7. **Icon (optional):** Additional icons to place inline with menu items.
+
+### Structure
+
+A profile menu is made up of two parts—an icon button and a popover. Profile
+menus support a variety of content that can range in complexity. Some profile
+menus are simple while other menus may be more complex and require multiple
+sections of information. Profile menus that are complex may benefit from
+including interactive elements depending on the use case.
+
+<Row>
+<Column colLg={12}>
+
+![Example of a simple and complex profile menu.](images/disclosures-profile-menu-simple-complex.png)
+
+</Column>
+</Row>
+
+<Caption>Example of a simple and complex profile menu.</Caption>
+
+### Functionality
+
+#### Mouse
+
+Clicking the icon button triggers the profile menu to open and reveals
+information related to the users profile and account. The user can then click on
+menu items and other interactive elements within the menu. To close the menu,
+click on the icon button trigger again or anywhere on the page out side of the
+menu.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a simple and complex profile menu.](images/disclosures-profile-menu-structure.png)
+
+</Column>
+</Row>
+
+#### Keyboard
+
+- `Tab` to the icon button trigger to apply focus.
+- To open the profile menu press `Enter` or `Space`.
+- To navigate the profile menu items press the `Up` and `Down` arrow keys.
+- To select an item in the profile menu press `Enter`.
+- To close the profile menu press `Enter`, `Space`, or `Esc` .
+
+### Best practices
+
+#### Use with a global toolbar header
+
+If a product has a global toolbar header and offers login capabilities use the
+profile menu. Provide a way for the user to log out of an account, make changes
+to profile information, or navigate to settings.
+
+#### Show information that is pertinent to the user account or session
+
+Include sections on a global contextual level that may be helpful to the user to
+see in the profile menu. For example, indicating the current plan or location
+with an option to edit or change the current information can be valuable to
+include in a profile menu depending on the use case.
+
+#### Add icons and links to help guide the user
+
+When appropriate, pair icons with related menu items or information to indicate
+the action that will be performed once it is clicked. Provide links next to
+related menu items or information to specify there are additional quick actions
+to take, whether it be upgrading an account plan or changing a location. Only
+add icons and links to a profile menu if completely necessary in order to not
+overwhelm the user and to reduce visual clutter within the menu.
+
+## Context menu
+
+A context menu is considered to be a disclosure pattern that is triggered by a
+right click interaction and opens a popover. A context menu is a defined list of
+options in a menu that appears above all other content on a page and is related
+to the selected content or element it was triggered from.
```

Comment:
I think the lack of a trigger button might make a context menu fall out of the disclosure distinction.

Reply:
So this is definitely true? If so, I am going to take this section for context menu out.